install.sh

```diff
@@ -0,0 +1,8 @@
+build-commands:
```

Comment:
I assume you wanted to add it to the main manifest?

Reply:
This is my first flathub application. I didn't know what I should add to the install.sh